.github/pr-labeler.config.yml

```diff
@@ -4,6 +4,11 @@ e2e:
       - changed-files:
           - any-glob-to-any-file: "packages/wrangler/e2e/**/*"
 
+c3-e2e:
+  - all:
+      - changed-files:
+          - any-glob-to-any-file: "packages/create-cloudflare/e2e-tests/**/*"
```

Comment:
How about also running whenever any files in `packages/create-cloudflare` change?

Reply:
[09f7cc2](https://github.com/cloudflare/workers-sdk/pull/8167/commits/09f7cc2eee6e4d16de0eaec3307ed9a8489a1f2d)